tests/run-travis.sh

```diff
@@ -37,7 +41,7 @@ if [ "$WP_TRAVISCI" == "phpunit" ]; then
 	echo "Running \`$WP_TRAVISCI\` with:"
 	echo " - $(phpunit --version)"
 	echo " - WordPress branch: $WP_BRANCH"
-	if [ "$WP_BRANCH" == "master" ]; then
+	if [ "master" == "$WP_BRANCH" ]; then
```

Comment:
bash ==! php ;) 

(not a concern at all)

Reply:
Yeah, maybe let's get rid of any Yoda before 🚢  Consistency FTW